tox.ini

```diff
@@ -6,14 +6,14 @@ envlist = py{37,38}-{linux,windows}
 python =
```

Comment:
I don't know why there's a tox config in this repo, we don't use it. @paddyroddy can you see any uses? Otherwise I think this can be deleted.

Reply:
no idea, I'll raise an issue

Reply:
https://github.com/brainglobe/cellfinder-core/issues/32